bindata/network/ovn-kubernetes/alert-rules-control-plane.yaml

```diff
@@ -66,3 +66,21 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: NorthBoundConnectionsError
+      annotations:
+        message: |
+          NorthBound DB has too many errors in connections.
```

Comment:
"OVN NBDB has a high rate of connection errors"

Reply:
Done.